### ISSUE TITLE
USB Timeout

### ISSUE THREAD

**Issue by EXO5KELETON** [Huntereb/Awoo-Installer#129]
Hi,

Have been using Awoo for a while sometime on previous and on latest version I get a USB timeout have tried different PC's different USB Leads etc.

Doesn't happen all the time but id say 6 out of 10 times I try it will happen.

Any ideas?

thanks for your work.

**Comment by impeeza** [comment]
Yes, read the documentation, is very clear: «do not use USB, Awoo is not meant to be used on USB»

**Comment by EXO5KELETON** [comment]
I have, but surely it’s just easier to fix the USB issue or is it not that easy. 

network installs are crap as way too slow even on a 5GHz network. And installing from SD card isn’t too good either even with a 512Gb Extreme card.

thank you for taking the time to comment not many devs on GitHub do.

Much appreciated.
